docs/faq-and-troubleshooting/common-questions/roblox-not-opening.md

```diff
@@ -5,35 +5,37 @@ icon: simple/roblox
 
 # :simple-roblox: Why is Roblox not opening?
 
-**You may be missing TPM 2.0 and Secure Boot. You can also make a new user account and move everything over.**
-
 Follow the methods below to fix the problem.
 
-=== "Method 1: Enabling TPM 2.0 and Secure Boot"
+1. Enable TPM and Secure Boot
 
-    !!! success ""
+    - To enable **TPM 2.0** and **Secure Boot**, you need to access the BIOS of your computer and enable **TPM 2.0** (or **Trusted Platform Module**) and **Secure Boot**.
 
-        If you're using Windows 11, you've already got TPM 2.0 and Secure Boot turned on (unless you bypassed). If that's the case, try [Method 2](#__tabbed_1_2).
+    Refer to your motherboard's manual if you don't know how to access your BIOS.
 
-    To enable **TPM 2.0** and **Secure Boot**, you need to access the BIOS of your computer and enable **TPM 2.0** (or **Trusted Platform Module**) and **Secure Boot**.
 
-    Refer to your motherboard's manual if you don't know how to access your BIOS.
+2. Enable UAC
+
+    - Open the Control Panel, search for `Change User Account Control settings` and set it to either Level 1, 2 or 3.
+      
+    If UAC was disabled (set to Level 0), reinstall Roblox after enabling it.
+
 
-=== "Method 2: Creating a New Local Account"
+3. Create a new Local Account
 
-    1. Navigate to the following section in the "Settings" app to create a new local account:
+    - Navigate to the following section in the "Settings" app to create a new local account:
 
```

Comment:
Unrelated, but I think this looks better formatting wise:
```

    - Navigate to the following section in the "Settings" app to create a new local account:
        
        - **:material-microsoft: Windows 11**
        
            `Accounts > Other users > Add other user`

        - **:material-microsoft-windows: Windows 10**
            
            `Accounts > Family & other users > Add someone else to this PC`
```

![image](https://github.com/Atlas-OS/docs/assets/65787561/8cb56e09-7ab0-4a94-83c9-b71564de7995)